README.md

```diff
@@ -77,3 +77,16 @@ Okay... why?
 * `COUNT(posts.deleted_at)` gives you the count of deleted posts, which is
   useful when writing a report. Define and use `Post.deleted.count` when you
   have Ruby available.
+
+Other Options
+-------------
+
+If you have a date or time column
+that does not follow the `attribute_at` convention,
+you can specify the attribute name:
```

Comment:
Can you:

1. Match the convention of wrapping at 80 chars
2. Mention that this is useful for date fields